### PR TITLE
Start APIService controller for API usage

### PR DESCRIPTION
**Pull request by orangedeng**
**Problem:**
The apiservice lister can't return any data by using get/list function.
The problem is that we don't start the apiservice controller
in user/controller.

**Solution:**
Enable APIService controller in user controller.

Related issue:
https://github.com/rancher/rancher/issues/20955

### CODE REVIEW

**Comment by gitlawr** [comment]
LGTM

**Comment by cjellick** [comment]
Can you explain in more detail how this fixes issue #20955? I do not see the direct connection

**Comment by orangedeng** [comment]
The issue @davidnuzik pointed out has been fixed by https://github.com/rancher/rancher/pull/21542. Then the newly created HPA will be `initializing` state instead of `active`. After the `AbleToScale` condition appears in HPA and is set `true`, the HPA state will be set active.
But when Logan validated this issue, he found out that https://github.com/rancher/rancher/issues/20955#issuecomment-516855423. The second problem is fixed by PR `https://github.com/rancher/types/pull/929` And this PR is going to fix the first problem.

**Comment by dramich** [comment]
This looks like a revert of https://github.com/rancher/rancher/commit/d9034c8e852eaf579f35360325aec6356c36da09#diff-1de1009377fbeb4507c945bdd07c8527. Why is this being changed back? Do all providers now support this API for the expected version?

**Comment by orangedeng** [comment]
All the version in GKE doesn't support `v2beta2.autoscaling` api group. So I need to disable hpa in the GKE cluster.

**Comment by orangedeng** [comment]
The reason we change back is in this PR message.
As we can't use api service controller in k3s 1.12.5-lite, the integration test will fail if we enable it. So we need to change back to the old way.

**Comment by cjellick** [comment]
Where is the integration test that you are talking about? What cluster is that test running against?

I don't feel comfortable with this change. @loganhz can you setup a call sometime this week with me, @orangedeng, & @dramich to discuss this? Note, I'm not available thursday afternoon

File: pkg/api/store/k8sversion/store.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	}
+}
+
+type Store struct {
+	manager       *clustermanager.Manager
+	clusterLister v3.ClusterLister
+	types.Store
+}
+
+func (s *Store) Watch(apiContext *types.APIContext, schema *types.Schema, opt *types.QueryOptions) (chan map[string]interface{}, error) {
```
Comment (cjellick) [inline pkg/api/store/k8sversion/store.go]: why do you only care about watch? Why does this not impact list or get?

**Comment by orangedeng** [comment]
As we discussed offline, we will hold this until Darren finished upgrading k3s to latest version.

**Comment by orangedeng** [comment]
Updated

**Pull request merged**